scripts/simulator_base.js

```diff
@@ -467,7 +467,7 @@ var SIMULATOR = {};
 				var target = alliedUnits[targets[key]];
 
 				// Check Nullify
-				if (target.nullified) {
+				if (target.nullified && !skill.ignore_nullify) {
```

Comment:
This change will need to be propagated to other nullify checks to future-proof from new BGEs that add this to other skill types.